pkg/controller/daemon/daemon_controller_test.go

```diff
@@ -545,7 +553,116 @@ func TestSimpleDaemonSetScheduleDaemonSetPodsLaunchesPods(t *testing.T) {
 			t.Fatalf("did not foud pods on nodes %+v", nodeMap)
 		}
 	}
+}
 
+// When ScheduleDaemonSetPods is enabled, DaemonSets without node selectors should
```

Comment:
Is this a copy/paste of another test's comment?

Reply:
done